frontend/libretro.c

```diff
@@ -2020,6 +2031,7 @@ static void update_variables(bool in_flight)
             "pcsx_rearmed_gpu_unai_fast_lighting",
             "pcsx_rearmed_gpu_unai_ilace_force",
             "pcsx_rearmed_gpu_unai_pixel_skip"
+            "pcsx_rearmed_gpu_unai_scale_hires"
```

Comment:
you need to increase the array size

frontend/libretro.c:2033:13: warning: initializer-string for array of chars is too long
 2033 |             "pcsx_rearmed_gpu_unai_pixel_skip"